builtins/packagedcode_msitools-linux/fetchcode_container.ABOUT

```diff
@@ -0,0 +1,12 @@
+about_resource: .
```

Comment:
You probably meant to use this as a base for msitools?

Reply:
Yes. I've created the file `packagedcode_msitools.ABOUT` and removed this file.